packages/functional-tests/src/tests/sound-test.ts

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRESDK from '@microsoft/mixed-reality-extension-sdk';
+import { Actor } from '@microsoft/mixed-reality-extension-sdk';
+import App from '../app';
+import delay from '../utils/delay';
+import destroyActors from '../utils/destroyActors';
+import Test from './test';
+
+export default class SoundTest extends Test {
+
+    constructor(app: App, private baseUrl: string) {
+        super(app);
+    }
+
+    public async run(): Promise<boolean> {
+        let success = true;
+
+        success = success && await this.runSoundTest();
+
+        return success;
+    }
+
+    public async runSoundTest(): Promise<boolean> {
+        // Make a root object.
+        const tester = MRESDK.Actor.CreateEmpty(this.app.context, {});
+
+        const textPromise = Actor.CreateEmpty(this.app.context, {
+            actor: {
+                name: 'label',
+                parentId: tester.value.id,
+                transform: {
+                    position: { x: 0, y: 2, z: 0 }
+                },
+                text: {
+                    anchor: MRESDK.TextAnchorLocation.MiddleCenter,
+                    color: { r: 30 / 255, g: 206 / 255, b: 213 / 255 },
+                    height: 0.3
+                }
+            }
+        });
+
+        const text = textPromise.value;
+        await textPromise;
+        text.text.contents = "Preloading Sound";
+
+        const soundId = this.app.context.assetManager.loadSound(
```

Comment:
Currently we use the "load" term to refer to calls that return AssetGroups, and "create" to refer to calls that return Assets. Not sure if we want to keep this, but that's what in place currently.

Reply:
That seems strange to me. "load" to me means getting from an external location, while "create" indicates generating from scratch. I guess that "create" to return asset may make sense if it's creating a handle for the asset out of something already loaded, but that's not the case here.

Reply:
Isn't that what "instantiate" calls were introduced to solve?

Reply:
I'm clarifying now that this is the soundasset

---
In reply to: [256979671](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/191#discussion_r256979671) [](ancestors = 256979671)